lib/CodeTransform/Tests/Adapter/WorseReflection/Refactor/fixtures/expandClass1.test

```diff
@@ -0,0 +1,11 @@
+// File: source
+<?php
+use SomeNamespace\Foo;
+
+echo 'Debugging ' . F<>oo::class;
+
+// File: expected
+<?php
+use SomeNamespace\Foo;
+
+echo 'Debugging ' . SomeNamespace\Foo::class;
```

Comment:
we need to add a namespaced test:

```
namespace Foo;
```

either in this case or in both cases the expanded class should be absolute `\SomeNamespace\Foo::class`

Reply:
True